organization/_posts/2019/2019-12-05-coordination.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "HSF Weekly Meeting #175, 5 December, 2019"
+layout: meetings
+---
+
+# {{page.title}}
+
+Present/Contributors: Caterina Doglioni, Graeme Stewart, David Lange, Eduardo Rodrigues, Chris Jones, Marco Clemencic, Daniel Elvira, Pere Mato, Benedikt Hegner, Witek Pokorski, Andrea Valassi, Heather Gray, Michel Jouvin, Ben Morgan, Liz Sexton-Kennedy, Gloria Corti, Kyle Knoepfel, Stefan Roiser, Marco Clemencic, Sudhir Malik, Adam Parker, James Walder, David Lange, Caterina Doglioni
```

Comment:
Caterina is written down twice ;-)

Reply:
They get so much done it's almost deserved!

Reply:
:-)!